Source package : ansible-lint
File name       : ansible-lint-24.7.0.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/ansible-lint/ansible-lint-24.7.0.tar.gz/sha512/434eff959ffcfdadeaddebcf1ce4a2ab63f09a5ac4d89412a17558899ef9614d0176794ebc6fce73c2db5fc97ba8f4ad1a334209acbde6ae6c44c1b65e08aa7d/ansible-lint-24.7.0.tar.gz
Source URL      : https://github.com/ansible/ansible-lint/archive/refs/tags/v24.7.0.tar.gz#/ansible-lint-24.7.0.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-ansible-lint-24.7.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 3072000
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-ansible-lint-24.7.0.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 3072000


diff -r source-ansible-lint-24.7.0.tar.gz-extracted/ansible-lint-24.7.0/.git_archival.txt lookaside-ansible-lint-24.7.0.tar.gz-extracted/ansible-lint-24.7.0/.git_archival.txt
3,4c3,4
< describe-name: v24.7.0
< ref-names: tag: v24.7.0
---
> describe-name: v24
> ref-names: HEAD -> main, tag: v24.7.0, tag: v24
